nipype/interfaces/io.py

```diff
@@ -513,7 +513,7 @@ def _fetch_bucket(self, bucket_name):
 
         # Init variables
         creds_path = self.inputs.creds_path
-        iflogger = logging.getLogger('interface')
+        iflogger = logging.getLogger('nipype.interface')
```

Comment:
I would actually remove this line, as well as the others (L590, L658), as they're overriding a file-level variable, but not changing it.

Reply:
nice catch, fixed.